DXMainClient/DXGUI/Generic/MainMenu.cs

```diff
@@ -954,5 +978,17 @@ public string GetSwitchName()
         {
             return "Main Menu";
         }
+
+        private void LaunchCampaignExe()
+        {
+            try
+            {
+                // Safety if user declines launching windows permission prompt
+                Process.Start(ProgramConstants.GamePath + ClientConfiguration.Instance.CampaignPath);
```

Comment:
I think this should be implemented as a part of `GameProcessLogic` with event handling and such; need @Rampastring's opinion on this.